### PR TITLE
WIP: Return mock value when fetching jwt in local mode

### PR DESCRIPTION
**Pull request by Robert-H-Leonard**
- [x] Add test to jwt in python sdk
- [x] Add test to jwt in typescript sdk

### CODE REVIEW

File: sdk/__test__/util.spec.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import axios from 'axios'
```
Comment (haseebrabbani) [inline sdk/__test__/util.spec.ts:1]: lets follow the existing unit test convention i.e. keep the unit test in the same folder as the file being tested (not in a new __test__ folder)

File: sdk/__test__/util.spec.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import { fetchJwt } from '../utils';
+
+jest.mock("axios");
+const mockedAxios = axios as jest.Mocked<typeof axios>;
```
Comment (haseebrabbani) [inline sdk/__test__/util.spec.ts:5]: will this end up using the real instance of axios? this line is just casting the real object (not creating a mock afaik). ideally we shouldnt import axios in unit tests

File: python-sdk/src/forta_agent/utils.py [hunk 3]
```diff
@@ -166,8 +167,9 @@ def fetch_jwt(claims, expiresAt=None) -> str:
 
     except requests.exceptions.RequestException as err:
         if("Name does not resolve" in str(err)):
+        ## If forta-jwt-provider can't be resolved that means the bot is not running inside of a node
             print("Could not resolve host 'forta-jwt-provider'. This url host can only be resolved inside of a running scan node")
-            raise err
+            return MOCK_JWT
```
[the remark this answers is not in the document]
Comment (Robert-H-Leonard) [reply python-sdk/src/forta_agent/utils.py]: Updated

File: sdk/utils.ts [hunk 4]
```diff
@@ -200,8 +201,10 @@ export const fetchJwt = async (claims: {}, expiresAt?: Date): Promise<{token: st
     const response = await axios.post(`http://${hostname}:${port}${path}`, data)
     return response.data
   } catch(err) {
+    // If forta-jwt-provider can't be resolved that means the bot is not running inside of a node
     if((err.message as string).includes("ENOTFOUND forta-jwt-provider")) {
-      throw Error("Could not resolve host 'forta-jwt-provider'. This url host can only be resolved inside of a running scan node") 
+      console.warn(`Detected bot running in local mode. Generating a mock JWT`)
+      return mockJwt;
```
[the remark this answers is not in the document]
Comment (Robert-H-Leonard) [reply sdk/utils.ts]: Updated

File: sdk/__test__/util.spec.ts (hunk 2 above)
Comment (Robert-H-Leonard) [reply sdk/__test__/util.spec.ts]: Updated to use mock

File: sdk/utils.ts [hunk 5]
```diff
@@ -173,10 +173,11 @@ export const getAlerts = async (query: AlertQueryOptions): Promise<AlertsRespons
   return response.data.data.alerts
 }
 
-export const fetchJwt = async (claims: {}, expiresAt?: Date): Promise<{token: string} | null> => {
+export const fetchJwt = async (claims: {}, expiresAt?: Date): Promise<string | null> => {
   const hostname = 'forta-jwt-provider'
   const port = 8515
   const path = '/create'
+  const mockJwt = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJib3QtaWQiOiIweDEzazM4N2IzNzc2OWNlMjQyMzZjNDAzZTc2ZmMzMGYwMWZhNzc0MTc2ZTE0MTZjODYxeWZlNmMwN2RmZWY3MWYiLCJleHAiOjE2NjAxMTk0NDMsImlhdCI6MTY2MDExOTQxMywianRpIjoicWtkNWNmYWQtMTg4NC0xMWVkLWE1YzktMDI0MjBhNjM5MzA4IiwibmJmIjoxNjYwMTE5MzgzLCJzdWIiOiIweDU1NmY4QkU0MmY3NmMwMUY5NjBmMzJDQjE5MzZEMmUwZTBFYjNGNEQifQ.9v5OiiYhDoEbhZ-abbiSXa5y-nQXa104YCN_2mK7SP0';
```
[the remark this answers is not in the document]
Comment (Robert-H-Leonard) [reply sdk/utils.ts]: Exporting this variable

File: sdk/__test__/util.spec.ts (hunk 1 above)
Comment (Robert-H-Leonard) [reply sdk/__test__/util.spec.ts]: Updated

**Review by haseebrabbani — APPROVED** [review]

**Pull request merged**